tool/gravity/cli/package.go

```diff
@@ -267,6 +269,11 @@ func executePackageCommand(s *localenv.LocalEnvironment, cmd string, loc loc.Loc
 		}
 	}
 
+	// Try and autofix sysctls / kernel modules during startup
+	if strings.ToLower(cmd) == "start" {
```

Comment:
`teleport` will also be `start`ed, so more context is needed I believe.

Reply:
I'm not sure about the full impact, but thinking about `teleport` being constantly restarted by systemd as it fails to connect as long as `gravity-site` is not up makes me uneasy.